app/working-with-app.md

```diff
@@ -372,57 +372,60 @@ port `8080` is allowed on the connection form your browser to your Docker host.
-as `install`, `upgrade`, `inspect` and `render` can be performed directly on applications while they are 
-stored in a registry.
+As mentioned in the introduction, `docker app` lets you manage entire
+applications the same way that you currently manage container images. For
+example, you can push and pull entire applications from registries like Docker
+Hub with `docker app push` and `docker app pull`. Other `docker app` commands,
+such as `install`, `upgrade`, `inspect` and `render` can be performed directly
+on applications while they are stored in a registry.
+
+Push the application to Docker Hub. To complete this step, you need a valid
+Docker ID and you must be logged in to the registry to which you are pushing
+the app.
+
+By default all platform architectures are pushed to the registry. If you are
+pushing an official Docker image as part of your app, you may find your app
+bundle becomes large with all image architectures embedded. To just push the
+architecture required you can add the `--platform` flag.
 
-The following section contains some examples.
-
-Push the application to Docker Hub. To complete this step, you need a valid Docker ID and you must be 
-logged in to the registry to which you are pushing the app.
-
-Be sure to replace the registry ID in the following example with your own.
+```bash
+$ docker login 
 
-```
-$ docker app push my-app --tag nigelpoulton/app-test:0.1.0
-docker app push hello-world.dockerapp --tag nigelpoulton/app-test:0.1.0
-docker.io/nigelpoulton/app-test:0.1.0-invoc
-hashicorp/http-echo
- application/vnd.docker.distribution.manifest.v2+json [2/2] (sha256:ba27d460...)
-<Snip>
+$ docker app push my-app --platform="linux/amd64" --tag <hub-id>/<repo>:0.1.0
 ```
 
-The app is now stored in the container registry.
-
 ### Push the app to DTR
 
-Pushing an app to Docker Trusted Registry (DTR) involves the same procedure as [pushing an app to Docker Hub](#push-the-app-to-docker-hub) except that you need your DTR user credentials and [your DTR repository information](/ee/dtr/user/manage-images/review-repository-info/). To use client certificates for DTR authentication, see [Enable Client Certificate Authentication](/ee/enable-client-certificate-authentication/).
+Pushing an app to Docker Trusted Registry (DTR) involves the same procedure as
+[pushing an app to Docker Hub](#push-the-app-to-docker-hub) except that you
+need your DTR user credentials and [your DTR repository
+information](/ee/dtr/user/manage-images/review-repository-info/). To use client
+certificates for DTR authentication, see [Enable Client Certificate
+Authentication](/ee/enable-client-certificate-authentication/).
+
+By default all platform architectures are pushed to DTR. If you are pushing an
+official Docker image as part of your app, you may find your app bundle
+becomes large with all image architectures embedded. To just push the
+architecture required you can add the `--platform` flag.
```

Comment:
I would apply the previous suggestions to this section.

Reply:
Fixed in #9170